tox.ini

```diff
@@ -1,23 +1,27 @@
 [tox]
-envlist = py{27,34,35,36}-dj{110,111}-wag{111,112},py36-flake8
+envlist = py36-dj200-wag{210,220,230,240},py36-dj210-wag{230,240},py36-flake8
```

Comment:
Was it the intention here to drop support for Python2?

Reply:
I think we have to for django 2